pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -75,10 +75,19 @@ const (
 	httpSource imageSource = "http"
 	// pvcSource defines the pvc source type for VM Disk Image.
 	pvcSource imageSource = "pvc"
-	// kubeVirtImagesNamespace namespace contains globally available custom images and cached standard images.
-	kubeVirtImagesNamespace           = "kubevirt-images"
-	dataVolumeStandardImageAnnotation = "kubevirt-initialization.k8c.io/standard-image"
-	osAnnotationForCustomDisk         = "cdi.kubevirt.io/os-type"
+	// DataVolumeStandardImageAnnotationKey is the annotation key for standard DataVolumes.
+	DataVolumeStandardImageAnnotationKey = "kubevirt-initialization.k8c.io/standard-image"
+	// DataVolumeStandardImageAnnotationKey is the annotation value for standard DataVolumes.
+	DataVolumeStandardImageAnnotationValue = "true"
+	// DataVolumeCustomDiskOsAnnotationKey is the annotation key for custom DataVolumes.
+	// Annotation value will be the os type.
+	DataVolumeCustomDiskOsAnnotationKey = "cdi.kubevirt.io/os-type"
+	// KubeVirtImagesNamespace namespace contains globally available custom images and cached standard images.
+	KubeVirtImagesNamespace = "kubevirt-images"
```

Comment:
I think we will not need that in machine controller as we pass  namespace/name if source of image is pvc.